guillotina_dbusers/services/auth.py

```diff
@@ -21,7 +21,7 @@ class Login(Service):
         data = await self.request.json()
         creds = {
             'token': data['password'],
-            'id': data['username']
+            'id': data['username'] or data['login']
```

Comment:
I think this should probably be `data.get('username', data.get('login'))`. Otherwise, it'll error if you don't provide `username`.